src/components/main/Main.js

```diff
@@ -22,8 +22,8 @@ export default function Main(props) {
 			<h3 className="main__title">Search results</h3>
 			{isPreloader? <Preloader />: <></>}
 			<ul className={`main__list${isPreloader?'_closed':''}`}>
-				{(isShowenMore?articles:showLessArray).map((article) => {
-					return <Article isHomePage={isHomePage} gettingSavedArticles={gettingSavedArticles} key={generateKey(article)} q={q} isLoggedIn={isLoggedIn} article={article} />;
+				{(isShowenMore?articles:showLessArray).map((article, idnex) => {
+					return <Article isHomePage={isHomePage} key={idnex} gettingSavedArticles={gettingSavedArticles} q={q} isLoggedIn={isLoggedIn} article={article} />;
```

Comment:
` key={index} `

It’s better to use `_id` of the card instead of the `index` for attribute `key`

Here is why we shouldn’t use `index` for that: [click here]( https://programmingwithmosh.com/react/why-do-i-need-keys-in-react-lists/#:~:text=React%20recommends%20that%20you%20do,to%20some%20unstable%20component%20behaviour.)

Reply:
it's better to correct this